src/commands/module/add.ts

```diff
@@ -116,28 +116,42 @@ async function updateNuxtConfig(
   rootDir: string,
   update: (config: any) => void,
 ) {
-  let _module: ProxifiedModule
-  const nuxtConfigFile = resolve(rootDir, 'nuxt.config.ts')
-  if (existsSync(nuxtConfigFile)) {
-    consola.info('Updating `nuxt.config.ts`')
-    _module = await loadFile(nuxtConfigFile)
+  const configFiles = [
+    'nuxt.config.ts',
+    'nuxt.config.js',
+    '.config/nuxt.ts',
+    '.config/nuxt.js',
+  ]
```

Comment:
Hi @Yizack , I have a question.

### Ques1

Is it possible to detect when `.config/` are nested?

https://github.com/pi0/config-dir?tab=readme-ov-file#nesting